packages/html/src/time-selector/index.ts

```diff
@@ -2,3 +2,6 @@ export * from './time-selector.spec';
 export * from './time-selector-column';
 export * from './time-selector-header';
 export * from './time-selector-fast-selection';
+export * from './templates/time-selector-all';
```

Comment:
how about `time-selector-normal` ?

Reply:
I am not sure, as there aren't any default columns; _time-selector-all_ defines all the possible columns